politeiawww/cmd/pictl/print.go

```diff
@@ -52,3 +55,53 @@ func byteCountSI(b int64) string {
 func printInPlace(s string) {
 	fmt.Printf("\033[2K\r%s", s)
 }
+
+// dollars converts an int64 price in cents into a human readable dollar
+// string.
+//
+// | Input     | Output          |
+// |-----------------------------|
+// | 13000     | "$130.00"       |
+// | 130000    | "$1,300.00"     |
+// | 13000000  | "$130,000.00"   |
+// | 130000000 | "$1,300,000.00" |
+// | 78        | "$0.78"         |
+// | -78       | "-$0.78"        |
+func dollars(cents int64) string {
+	// get the value in dollars
```

Comment:
The comment style of the politeia and politeaigui repos are slightly different.  When contribuing to the politeia repo, please capitalize the first letter of the comment.
```suggestion
	// Get the value in dollars
```

Apply to all comments.